docs/operating-scylla/_common/tools_index.rst

```diff
@@ -6,8 +6,6 @@
 * :doc:`ScyllaDB SStable </operating-scylla/admin-tools/scylla-sstable>` - Validates and dumps the content of SStables, generates a histogram, dumps the content of the SStable index.
```

Comment:
These places may need to be updated:
https://github.com/scylladb/scylladb/blob/2e33234e911f0e9e6a1c157aefd1f911e0cb1ee5/docs/operating-scylla/diagnostics.rst?plain=1#L106
https://github.com/scylladb/scylladb/blob/2e33234e911f0e9e6a1c157aefd1f911e0cb1ee5/docs/troubleshooting/pointless-compactions.rst?plain=1#L28